src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -578,16 +578,14 @@ void SolverInterfaceImpl::resetMesh(
     int meshID)
 {
   PRECICE_TRACE(meshID);
+  PRECICE_ASSERT(false, "At the moment, we can no longer and not yet again reset meshes.");
```

Comment:
- Why this assertion of `false` here?
- What does `we can no longer and not yet again` mean?

I assume this was for debugging purposes and should be removed, right?
```suggestion
```

Reply:
Currently, `resetMesh` is not allowed to be called and also not reachable from `SolverInterface`. 
"We can no longer and not yet again" means that this function was needed for dynamic meshes as implemented by Bernhard, for which we dropped support for the sake of simpler parallelization. And it will presumably be needed again once we have "parallel" dynamic meshes.

Reply:
So, the assertion wants to say "we shouldn't be here in the current preCICE version".

The "no longer" and "not yet again" I really understood as time of the simulation, not version on preCICE. Maybe this reply would be a nice comment to add next to the assertion.

Reply:
I removed the assertion altogether as still some tests depend on this function. Will be resolved as part of #225